DOCSP-36189 Add custom error handling for triggers with AWS EventBridge (#674)
mongodb/docs-app-services: source/functions/handle-errors.txt

This page explains how to work with errors in Atlas Functions.

> .. note:: Custom Error Handling for Triggers Using EventBridge

Reviewer: ```suggestion
.. note:: Custom Error Handling for Database Triggers Using AWS EventBridge
```

>   You can create a custom error handler function specifically for 

Reviewer: ```suggestion
  You can create a function specifically for handling errors in
```

>   Triggers using EventBridge. For more information, refer to

Reviewer: ```suggestion
  database Triggers using AWS EventBridge. For more information, refer to
```
